cmd/olm/cleanup_test.go

```diff
@@ -540,3 +599,12 @@ func newRoleBinding(namespace, name string, uid types.UID, ownerRefs ...metav1.O
 	roleBinding.SetOwnerReferences(ownerRefs)
 	return roleBinding
 }
+
+func newAPIService(name string, uid types.UID, labels map[string]string, ownerRefs ...metav1.OwnerReference) *apiregv1.APIService {
```

Comment:
Could you explain how the `labels` and `ownerReferences` come into play in this bug in the commit?